app/src/main/java/com/xmartlabs/daydreaming/ui/TrendingScreenFragment.java

```diff
@@ -0,0 +1,139 @@
+package com.xmartlabs.daydreaming.ui;
+
+import android.os.Bundle;
+import android.support.annotation.Dimension;
+import android.support.annotation.LayoutRes;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.LinearLayout;
+import android.widget.ScrollView;
+
+import com.annimon.stream.Optional;
+import com.hannesdorfmann.fragmentargs.annotation.FragmentWithArgs;
+import com.xmartlabs.daydreaming.R;
+import com.xmartlabs.daydreaming.helper.ui.MetricsHelper;
+
+import butterknife.BindView;
+
+/**
+ * Created by chaca on 4/13/17.
+ */
+@FragmentWithArgs
+public class TrendingScreenFragment extends BaseFragment {
+  @Dimension(unit = Dimension.PX)
+  private static final int BLACK_LINE_HEIGHT = MetricsHelper.dpToPxInt(3);
+  private static final int QUANTITY_OF_OPTIONS = 3; //TODO change it for a list
+
+  @BindView(R.id.animals_option_view)
+  DiagonalLayoutView animalsOptionView;
+  @BindView(R.id.art_option_view)
+  DiagonalLayoutView artOptionView;
+  @BindView(R.id.bottom_gray_diagonal_view)
+  DiagonalLayoutView bottomGrayView;
+  @BindView(R.id.fifth_black_line_view)
+  DiagonalLayoutView fifthBlackLineView;
+  @BindView(R.id.first_black_line_view)
+  DiagonalLayoutView firstBlackLineView;
+  @BindView(R.id.food_option_view)
+  DiagonalLayoutView foodOptionView;
+  @BindView(R.id.fourth_black_line_view)
+  DiagonalLayoutView fourthBlackLineView;
+  @BindView(R.id.nature_option_view)
+  DiagonalLayoutView natureOptionView;
+  @BindView(R.id.scroll_view)
+  ScrollView scrollView;
+  @BindView(R.id.second_black_line_view)
+  DiagonalLayoutView secondBlackLineView;
+  @BindView(R.id.sports_option_view)
+  DiagonalLayoutView sportsOptionView;
+  @BindView(R.id.third_black_line_view)
+  DiagonalLayoutView thirdBlackLineView;
+  @BindView(R.id.toolbar)
+  Toolbar toolbarView;
+  @BindView(R.id.travel_option_view)
+  DiagonalLayoutView travelOptionView;
+
+  @LayoutRes
+  @Override
+  protected int getLayoutResId() {
+    return R.layout.trending_option_fragment;
+  }
+
+  @Override
+  public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+    super.onViewCreated(view, savedInstanceState);
+    setUpView(view);
+    setupToolbar();
+  }
+
+  private void setUpView(View view) {
+    view.post(() -> {
+      @Dimension(unit = Dimension.PX)
```

Comment:
Check if the view is still alive in the callback. Remember that this is a common issue. Try running the monkey to look for this kind of errors too.